Source/KeyboardManager.swift

```diff
@@ -5,35 +5,51 @@
 
 import UIKit
 
-public typealias KeyboardManagerEventClosure = (KeyboardManagerEvent, KeyboardManagerEventData) -> Void
+public typealias KeyboardManagerEventClosure = (KeyboardManagerEvent) -> Void
 
 public enum KeyboardManagerEvent {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/alphatroya/KeyboardManager/blob/8c0082f605e3eb6bffa77060ffae9f2433310bd7/Source/KeyboardManager.swift#L10"></span>Public declarations should be documented.</td>
    </tr>
  </tbody>
</table>